### DISCUSSION TITLE
whisper use in  word level timestamps

### DISCUSSION THREAD

**Discussion by li-henan** [openai/whisper#1496] [General]
Dear author, could you help me to solve this problem?
I input :whisper p226_007.wav --model tiny --device cpu --word_timestamps True , cause I want to see word level timestamps, but errors occur like this : AssertionError: 3D tensors expect 2 values for padding , is my python version wrong?
Thanks very much for replying

**Comment by li-henan** [comment]
could you help me ，friends who research whisper ，thanks a lot

**Reply by MrCary** [reply]
use the torch=2.0.1,it will fix this bug
